miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/file/MiniAppFileChooser.kt

```diff
@@ -66,26 +69,33 @@ class MiniAppFileChooserDefault(var requestCode: Int) : MiniAppFileChooser {
         val clipData = intent.clipData
         when {
             data != null -> {
-                callback?.onReceiveValue((arrayOf(data)))
+                this.callback?.onReceiveValue((arrayOf(data)))
             }
             clipData != null -> {
                 val uriList = mutableListOf<Uri>()
                 for (i in 0 until clipData.itemCount) {
                     uriList.add(clipData.getItemAt(i).uri)
                 }
 
-                callback?.onReceiveValue((uriList.toTypedArray()))
+                this.callback?.onReceiveValue((uriList.toTypedArray()))
             }
             else -> {
-                callback?.onReceiveValue(null)
+                this.callback?.onReceiveValue(null)
             }
         }
+        resetCallback()
     }
 
     /**
      * Can be used when HostApp wants to cancel the file choosing operation.
      */
     fun onCancel() {
-        callback?.onReceiveValue(null)
+        this.callback?.onReceiveValue(null)
```

Comment:
adding this is redundant :)

Reply:
Oh, we need it in order to resetting the values to `null` while cancelling operation e.g. when users want to press back button to return to the root state.